angular-users/src/app/components/user-list/user-list.component.ts

```diff
@@ -9,29 +11,48 @@ import { UserService } from 'src/app/services/user.service';
 export class UserListComponent implements OnInit {
 
   users: any;
+  professions: any;
   currentUser = null;
   currentIndex = -1;
+  currentProfessions = null;
+  currentIndexProfessions = -1;
   firstName = '';
-  message = 'Are you sure to delete all users';
+  delete_message = 'Are you sure you want to delete the user';
```

Comment:
Why currentIndexProfessions is in camel case and delete_message is in undescore case?